### DISCUSSION TITLE
Do not load cookie consent in iframe popup

### DISCUSSION THREAD

**Discussion by bennetsadyba** [orestbida/cookieconsent#636] [General]
Hi,

in my environment when I load a HTML page a "fancybox" iframe pops up (same domain). What happens is a cookie consent banner pops up on the HTML page and a second one inside the iframe. Do you have a solution for that? Exclude loading cookie consent in iframe, maybe? Any hints as to the code?

**Comment by orestbida** [comment]
I'm not sure what the relation between the html pages is. Are you loading the same html page in the iframe? I would need to see your project setup to give you any meaningful feedback.
